testMVVMTests/MVVMTestProjectTests.swift

```diff
@@ -12,7 +12,11 @@ import PromiseKit
 let mockupSeasons = [Season(name: "blah", episodes: [Episode(name: "episode1"), Episode(name: "episode2"), Episode(name: "episode3")])]
 
 class Test1SeasonServices: SeasonsServices {
-
+    internal func update(episode: Episode, name: String?) -> Promise<Episode> {
+        return Promise { fulfill, reject in
```

Comment:
Well, this was just a quick fix of already failing test so I used the same solution as in other test cases. This is imho out of the scope of the PR, feel free to improve the test cases.